CHANGES_NEXT_RELEASE

```diff
@@ -17,6 +17,7 @@
 - Add: Support for null values in string filters (#2359)
 - Fix: Partial sanity check for custon url, only if the url contains no replacements using ${xxx} (#2280, #2279)
 - Fix: if entity::type is given but as an empty string, in payload of POST /v2/subscriptions or PATCH /v2/subscriptions/{sub}, an error is returned (#1985)
+- Fix: more intelligent, quote-aware, operator lookup inside q/mq string filters (#2452)
```

Comment:
Sure.
[ I didn't really see this fix as a 'fix for 2452', it is much more. #2452 just happens to get a way to work thanks to this PR (if quotes are used, of course) ]